src/index.ml

```diff
@@ -62,6 +62,8 @@ let may f = function None -> () | Some bf -> f bf
 
 exception RO_Not_Allowed
 
+exception Different_Sizes_Not_Allowed
```

Comment:
What would you think of having exceptions like
```
exception Invalid_Key_Size of key
exception Invalid_Value_Size of value
```
to get better feedback ?

Reply:
Seems good to me. Would also make the `check_raises` tests slightly more precise.